samples/positive/development/potentially_compromised_attacker_sample.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Version": "development",
+  "ReporterInfo": {
+    "ReporterOrg": "ExampleOrg",
+    "ReporterOrgDomain": "example.com",
+    "ReporterOrgEmail": "reports@example.com",
+    "ReporterContactEmail": "contact@example.com",
+    "ReporterContactName": "Mr. Example",
+    "ReporterContactPhone": "+ 01 000 1234567"
+  },
+  "Disclosure": true,
+  "Report": {
+    "ReportClass": "Activity",
+    "ReportType": "PotentiallyCompromisedAccount",
+    "Date": "2018-02-05T14:17:10Z",
+    "DestinationIp": "198.51.100.33",
+    "DestinationPort": 80,
+    "Ongoing": true,
+    "Account": {
+      "AccountName": "Karl Heinz",
+      "AccountNumber": 89789778643782
```

Comment:
Schema contains string-based `AccountIdentifier` that is not reflected in any positive samples. Instead number based `AccountNumber` is included in the samples but not in the schema.

Reply:
@FrederikP I've updated the test files and added another negative case for using `AccountIdentifier` instead of `AccountEmail`.